package.json

```diff
@@ -5,8 +5,12 @@
   "private": true,
   "scripts": {
     "dev": "NODE_ENV=development DEBUG=metalsmith-timer,metalsmith-algolia,metalsmith-layouts,metalsmith-assets node index.js",
-    "build": "NODE_ENV=production DEBUG=metalsmith-timer,metalsmith-algolia node index.js",
-    "webpack": "webpack --progress --colors"
+    "build": "DEBUG=metalsmith-timer,metalsmith-algolia node index.js",
+    "webpack": "webpack --progress --colors",
+    "build:production": "NODE_ENV=production npm run build # no drafts, no beta, no archive",
+    "build:preview": "NODE_ENV=preview npm run build # includes drafts, beta and archive",
+    "build:beta": "NODE_ENV=beta npm run build # includes beta, no drafts, no archive",
+    "build:archive": "NODE_ENV=archive npm run build # all production and archive"
```

Comment:
I think archiving is out of scope for this, but good to know we can do it. I think we'd still be okay simply cutting a tag when we want to archive. That way we get to remove all those docs from the repo and keep it fast